src/backscratcher/StrategyProxy.sol

```diff
@@ -194,6 +200,7 @@ contract StrategyProxy {
         require(thisStake.liquidity != 0, "kek_id not found");
 
         if (thisStake.liquidity > 0) {
+            proxy.safeExecute(_gauge, 0, abi.encodeWithSignature("withdrawLocked(bytes32,address)", _kek_id, address(proxy)));
```

Comment:
Doesn't this break in the case where the gauge's `withdrawLocked` method only accepts a single `kek_id` argument, like with FRAX/USDC?

Reply:
No, because this one is "withdrawV2", the FRAX/DAI and FRAX/USDC ones use "withdrawV3"